packages/block-library/src/columns/block.json

```diff
@@ -28,7 +28,7 @@
 			}
 		},
 		"spacing": {
-			"blockGap": true,
+			"blockGap": [ "vertical", "horizontal" ],
```

Comment:
Just a note that I think this is a deceptively powerful feature (by which I mean, very cool feature!) — because vertical spacing only affects the columns block on mobile viewports, this means that it creates a nice ability to be able to control desktop versus mobile spacing, without having to consider viewport widths directly 👍

Reply:
> Which formats do we support here and what do they mean?

I'm not clear what you mean here. Was it in relation to the [above comment](https://github.com/WordPress/gutenberg/pull/37736#discussion_r804376196), or the formats of `"blockGap"` in `theme.json` (`string|array`)?

> because vertical spacing only affects the columns block on mobile viewports, this means that it creates a nice ability to be able to control desktop versus mobile spacing, without having to consider viewport widths directly

This comment makes me think of a recent conversation on how blockGap effects margin-top via the layout abstraction. Namely, how can we reconcile the need for zero margins in a world where `"blockGap"` has a value in theme.json.

https://github.com/WordPress/gutenberg/pull/37344